src/providers/mbed_provider/key_management.rs

```diff
@@ -190,7 +189,7 @@ impl MbedProvider {
 
         let id = key::Id::from_persistent_key_id(key_id);
         let key_attributes = key::Attributes::from_key_id(id)?;
-        let buffer_size = key_attributes.export_key_output_size()?;
+        let buffer_size = key_attributes.export_public_key_output_size()?;
```

Comment:
Thought I'd point out, this change doesn't actually affect anything. `export_key_output_size` and `export_publid_key_output_size` both call the same function in `psa-crypto`. But since we have a `public` version, I thought we should use it 😉.